cli/tests/depcheck.sh

```
@@ -1,12 +1,15 @@
+if [ -d "output/depcheck" ]; then
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)